addons/website/models/ir_http.py

```diff
@@ -346,7 +346,9 @@ def _serve_fallback(cls, exception):
 
         redirect = cls._serve_redirect()
         if redirect:
-            return request.redirect(_build_url_w_params(redirect.url_to, request.params), code=redirect.redirect_type)
```

Comment:
Perhaps we should add the comment about the safety of the `local=False` here, to avoid copy/paste mistakes (or at least to give it a chance)?